symplyphysics/laws/gravity/vector/falling_body_displacement.py

```diff
@@ -0,0 +1,102 @@
+from sympy import Expr, Rational
+from symplyphysics import (
+    angle_type,
+    units,
+    Quantity,
+    Vector,
+    QuantityVector,
+    validate_input,
+    validate_output,
+    scale_vector,
+    add_cartesian_vectors,
+    cross_cartesian_vectors,
+)
+
+# Description
+## Suppose a reference frame S' is fixed to a moving object A (e.g., Earth) and some body B moving freely
+## (i.e. the sum of external non-gravitational forces acting on it is zero). In the case of an inertial
+## frame of reference, the displacement of body B from the starting position would follow the usual rule
+## `s = v0 * t + (g / 2) * t**2`. But in the case of non-inertial frames, we have to take the Coriolis and
+## the centrifugal force into account as well, which results into the following series:
+
+# Conditions
+## - The sum `F` of all other, non-gravitational forces acting on body B is 0.
+## - `g` is independent of coordinates.
+
+# Notes
+## - Note that the series is truncated at the fifth term. More terms can be obtained by plugging the result
+##   into the equation of motion `a = g + cross(v, w)` and integrating it over time.
+
+# Law: s = v0 * t
+#        + t**2 * (g / 2 + cross(v0, w))
+#        + t**3 / 3 * (cross(g, w) + 2 * cross(cross(v0, w), w))
+#        + t**4 / 6 * cross(cross(g, w), w)
+#        + O(t**5)
+## t - time
+## s - vector of displacement of body B in S'
+## v0 - vector of initial velocity of body B
+## w - pseudovector of angular velocity of rotation of moving frame S'
+## g - vector of acceleration due to gravity (or free fall acceleration) of body B
+## cross(a, b) - cross product between vectors `a` and `b`
+## O - [big O](https://en.wikipedia.org/wiki/Big_O_notation)
```

Comment:
There are two different applications for big O notation. Put small explanation here as well.